scripts/subfolder_list.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+
+def touch(trigger):
+    if(trigger is None):
+        return
+    if os.path.exists(trigger):
+        print("Touching")
```

Comment:
This needs to either be removed, or be more verbose.

Reply:
actually the print message was for debugging, and should have been removed.
Will add comment on why to retrurn.

Reply:
It is not necessary to run ninja twice.
The first invocation will detect the new files and sys call are parsed correctly.
However, before next invocation of ninja, we need to ensure the header dependencies are updated in case they will be modified.
So therefore CMake is invoked on second ninja.

Reply:
"It is not necessary to run ninja twice."

My mistake.